### PR TITLE
fix: correct typo in notification template

### PR DESCRIPTION
**Pull request by mtojek**
Related: https://github.com/coder/coder/pull/14010

There is a typo in the notification template, unescaped character.

### CODE REVIEW

File: coderd/database/migrations/000234_fix_notifications_user_created.down.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+UPDATE notification_templates
```
Comment (dannykopping) [inline coderd/database/migrations/000234_fix_notifications_user_created.down.sql:1]: I don't really see value in restoring the typo in the down migration.

File: coderd/database/migrations/000234_fix_notifications_user_created.up.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+UPDATE notification_templates
+SET
+    body_template = E'Hi {{.UserName}},\n\nNew user account **{{.Labels.created_account_name}}** has been created.'
```
Comment (dannykopping) [inline coderd/database/migrations/000234_fix_notifications_user_created.up.sql:3]: This was incorrectly set in the [initial migration](https://github.com/coder/coder/blob/83963b9e610c58a99c8c516e8eeb00a37a428ed9/coderd/database/migrations/000233_notifications_user_created.up.sql#L4), let's sneak it in here if you don't mind

```suggestion
    body_template = E'Hi {{.UserName}},\n\nNew user account **{{.Labels.created_account_name}}** has been created.',
    group = 'User Events'
```

**Review by dannykopping — APPROVED** [review]

File: coderd/database/migrations/000234_fix_notifications_user_created.down.sql (hunk 1 above)
Comment (mtojek) [reply coderd/database/migrations/000234_fix_notifications_user_created.down.sql:1]: just consistency?

File: coderd/database/migrations/000234_fix_notifications_user_created.up.sql (hunk 2 above)
Comment (mtojek) [reply coderd/database/migrations/000234_fix_notifications_user_created.up.sql:3]: I would rather keep this idempotent. It does not seem to be an urgent fix that requires modifying existing files, and if there are some scripts checking the content of migration files, and I wouldn't like to make them suffer.

Comment (dannykopping) [reply coderd/database/migrations/000234_fix_notifications_user_created.up.sql:3]: Maybe you missed the content of the change? The event is a user event, but its group is incorrectly set to `Workspace Events` currently.

Comment (mtojek) [reply coderd/database/migrations/000234_fix_notifications_user_created.up.sql:3]: Oh crap, let me redo it.

Comment (mtojek) [reply coderd/database/migrations/000234_fix_notifications_user_created.up.sql:3]: https://github.com/coder/coder/pull/14112

**Pull request merged**